go/ec2/README.md

```diff
@@ -0,0 +1,202 @@
+# AWS SDK for Go code examples for Amazon Elastic Compute Cloud (Amazon EC2)
```

Comment:
...for **Amazon EC2**
(use short version in title)

Reply:
Fixed